js-intro.js

```diff
@@ -4,58 +4,195 @@
 // Please read all questions thoroughly
 // Psuedo coding is highly recommended
 
+
+//-------------------------------------------------------------------------
+
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
+
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+
+/*
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
+// Creating boiling point temp
+var tempBoil = 212
+
+//Creating a series of functions for each temp
+
+const isItBoiling1 = (butseriouslyisit) => {
+    if(temp1 < tempBoil ) {
+        return `${temp1} is below the boiling point`
+    }
+}
+
+const isItBoiling2 = (butseriouslyisit) => {
+    if(temp2 > tempBoil ) {
+        return `${temp2} is above the boiling point`
+    }
+}
+
+const isItBoiling3 = (butseriouslyisit) => {
+    if(temp3 || tempBoil ) {
+        return `${temp3} is at the boiling point`
+    }
+}
 
+//Logging the results to console.
+//Expected result ----> 35 is below the boiling point
+350 is above the boiling point
+212 is at the boiling point
+console.log(isItBoiling1())
+console.log(isItBoiling2())
+console.log(isItBoiling3())
 
 
+*/
 
-// --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
-// Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
+//-------------------------------------------------------------
 
+
+// // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
+// // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
+/*
 var myNumbers1 = [3, 7, 0, 6, -9]
 
 
+//Creation of multiply function
+const multiplyBy = (multiplied) => {
```

Comment:
Same as above: your function in it's current state isn't reusable and can only tell us what `myNumbers1` * 5 will be. If we wanted to check another array of numbers * 5, we would have to rewrite this function. Consider using your argument `multiplied` on the next lines in place of `myNumbers1`, then invoke the function with an argument.

Reply:
Understood, yet again! Thanks... yet again!